Source/WebCore/dom/DocumentMarker.h

```diff
@@ -38,7 +38,7 @@ namespace WebCore {
 // It optionally includes a description that could be displayed in the user interface.
 class DocumentMarker : public CanMakeWeakPtr<DocumentMarker> {
 public:
-    enum MarkerType {
+    enum class Type : uint32_t {
```

Comment:
Why aren't we using a uint16_t?

Reply:
People can easily change the type when they add enum types.

Reply:
We should always use typing as tight as possible. A uint16_t can be packed with a compact pointer while a uint32_t couldn't for e.g.